.github/workflows/docker.yaml

```diff
@@ -0,0 +1,26 @@
+on:
+  release:
+    types: [created]
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build Image
+        id: build-image
+        uses: redhat-actions/buildah-build@v2
+        with:
```

Comment:
The current container image on Docker hub supports architectures:

```
linux/amd64
linux/arm/v7
linux/arm64
```

I'm not sure whether this workflow accounts for that at the moment. Do we need to incorporate `archs` to the `redhat-actions/buildah-build@v2` action? Similar to the [example](https://github.com/redhat-actions/buildah-build/blob/main/.github/workflows/multiarch.yml) 🙂

Reply:
Yes, we need to. Good spot, I will update PR